### DISCUSSION TITLE
How to restrict automatic tab with docs?

### DISCUSSION THREAD

**Discussion by anton-magurskyi** [Authenticator-Extension/Authenticator#1192] [General]
Hi! I'm trying to use Authenticator with pytest (pyotp). and besides my main page, I always get an additional tab opened (leading to https://authenticator.cc/docs/en/chrome-issues). Please help me to turn this feature off.

**Comment by mymindstorm** [comment]
Please use a TOTP library like [pyotp](https://pypi.org/project/pyotp/) instead of a browser extension for this.

**Reply by anton-magurskyi** [reply]
lol! I had no idea it was possible to use them separately. I spent a few days putting the extension into the playwright's internal Chromium browser. OMG. Thanks a lot for the clarification!
